virttest/utils_test/libvirt.py

```diff
@@ -922,6 +922,23 @@ def get_parts_list(session=None):
     return parts
 
 
+def get_disk_by_serial(session, serial_str):
```

Comment:
this method should reside in virttest/utils_disk.py

Reply:
@balamuruhans ,If move this method to virttest/utils_disk.py, there will be a circular imports.
Do you think we must move it to virttest/utils_disk.py?

Reply:
@Yingshun, I think we should keep the methods in the respective files. All the disk related methods should be placed in utils_disk.
Currently utils_misc and utils_libvirt is growing and needs to be reorganised by moving the methods to appropriate files. 

Can you move the disk related methods from libvirt.py to utils_disk.py so that this circular import situation can be fixed. `get_disk_by_serial()` calls `get_parts_list()`and if we place in utils_disk.py, then there wont be circular import issue ?

Reply:
@balamuruhans  Updated. Please help review again. Thanks very much!